proto/internal/temporal/server/api/replication/v1/message.proto

```diff
@@ -58,6 +58,7 @@ message ReplicationTask {
     // The task_type + data provide more flexibility in future use cases.
     temporal.api.common.v1.DataBlob data = 12;
     google.protobuf.Timestamp visibility_time = 9;
+    int32 priority = 13;
```

Comment:
maybe just live-traffic and backfill-traffic enums?

Reply:
Synced offline, will keep high/low priority.